sub-logs/studiodev.json

```diff
@@ -0,0 +1,19 @@
+{
+    "owner": {
+        "username": "ashfaq1994",
+        "email": "tricksfree.in@gmail.com"
+    },
+
+    "target": {
+        "A": {
+            "name": "studiodev.is-a-good.dev",
```

Comment:
name should only be studiodev

Reply:
Update the package.json, Please check and let me know.